packages/web-worker/README.md

```diff
@@ -67,9 +67,9 @@ self.onmessage = (e) => {
 ```
 
 ```ts
+import MyWorker from '../worker?worker'
 // worker.test.ts
 import '@vitest/web-worker'
-import MyWorker from '../worker?worker'
```

Comment:
This file is still fixed after disabling `**/*.md`. Let me check this further.

Reply:
Took a while to figure out weird globbing `**/*.md/${GLOB_SRC}`  :sweat_smile: https://github.com/vitest-dev/vitest/pull/6751/commits/f32bdbbb9f55ac2a2a622ac4727a8ef6df1d6ac6